### PR TITLE
Add the celery boilerplate

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  iib-worker:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile-workers
+    # Override the default command so that Celery auto-reloads on code changes
+    command:
+    - /bin/bash
+    - -c
+    - >-
+      pip3 install watchdog[watchmedo] &&
```
Comment (yashvardhannanavati) [inline docker-compose.yml]: shouldn't it just be `pip3 install watchdog`? Am I missing something?

Comment (yashvardhannanavati) [reply docker-compose.yml]: Ah, got it. nvm :)

Comment (mprahl) [reply docker-compose.yml]: By doing `watchdog[watchmedo]`, it installs `watchdog` with the extra requirements required to use the `watchmedo` feature.

You can see that this is defined here:
https://github.com/gorakhargosh/watchdog/blob/b752e371792eadf4d1e79cdcc15c40f3539bcb96/setup.cfg#L7-L10

Comment (yashvardhannanavati) [reply docker-compose.yml]: Ah, I learned something today. thanks!

**Comment by yashvardhannanavati** [comment]
:+1:

**Pull request merged**